users/models.py

```diff
@@ -9,3 +10,13 @@ class Profile(models.Model):
 
     def __str__(self):
         return f'{self.user.username} Profile'
+
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+
+        img = Image.open(self.image.path)
+
+        if img.height > 300 or img.width > 300:
```

Comment:
what about testing this change on images saved to the database?

Reply:
+1 on that

Reply:
I added a test to the image size